_docs-sources/guides/build-it-yourself/achieve-compliance/deployment-walkthrough/manual_steps.md

```diff
@@ -0,0 +1,121 @@
+---
+pagination_label: Deployment Walkthrough
+---
+
+# Manual steps
+
+Now that your infrastructure is up and running, there are some manual steps necessary before the resources meet the CIS AWS Benchmark recommendations.
+
+## Revoke Gruntwork's access
+
+Now that your infrastructure is deployed, Gruntwork doesn't need access to it anymore. Use the `gruntwork` CLI to revoke the access, [following these steps](https://github.com/gruntwork-io/gruntwork#revoking-access-to-aws):
```

Comment:
What does revoking our access actually do? Should we mention that our access is enabled through the `GruntworkAccessRole` in each of their accounts - and that this revoke command will delete the role in each of their accounts?

Reply:
Good point! I will add this.